creator-node/prod.env

```diff
@@ -6,6 +6,7 @@ dbConnectionPoolMax=500
 redisHost=cache
 redisPort=6379
 
+audius_enable_rsyslog=false
```

Comment:
I think this one is supposed to be camelCase while discovery is snake_case

Thanks for the comment separation

Reply:
I did notice that. Why do we do camelCasing in creatorNode and not in discovery_provider?

And sure thing!

Reply:
Because creator node is written in java script and discovery provider is written in python